WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditAddressView.swift

```diff
@@ -0,0 +1,166 @@
+import Foundation
+import Combine
+import SwiftUI
+
+/// Hosting controller that wraps an `EditAddressView`.
+///
+final class EditAddressHostingController: UIHostingController<EditAddressView> {
+
+    init() {
+        super.init(rootView: EditAddressView())
+    }
+
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+/// Allows merchant to edit the customer provided address of an order.
+///
+struct EditAddressView: View {
```

Comment:
There are some examples on Apple's side like `ContentView`, `MapView`, but I agree with you and don't like overuse of `View` and `ViewController` naming.
Renamed to `EditAddressForm` since it describes function better in 506b12a7cfe4085c3788b325f8c321f3bb3cd66a